.gitignore

```diff
@@ -4,6 +4,8 @@
 *~
 /examples/*/node_modules/
 
+/.idea
```

Comment:
@SimenB we care to remove this or just leave it alone?

Reply:
I've got it in my global gitignore. doesn't hurt, though?

Reply:
Doesn't hurt. People have opinions on the topic (local vs global gitignore), but it's not a big deal to me too so meh 🤷‍♂️ .